tests/network/localnet/migration_stuntime/conftest.py

```
@@ -0,0 +1,123 @@
+    localnet_vm,
+)
+from tests.network.localnet.migration_stuntime import libstuntime
+from utilities.infra import get_node_selector_dict
+
+LOGGER = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope="class")
+def localnet_stuntime_server_vm(
+    unprivileged_client: DynamicClient,
+    nncp_localnet_on_secondary_node_nic: libnncp.NodeNetworkConfigurationPolicy,
```

Ruff (0.15.10)

[warning] Unused function argument: `nncp_localnet_on_secondary_node_nic`

(ARG001)